audit-log-explorer/README.md

```diff
@@ -0,0 +1,78 @@
+# Importable Audit Log Explorer App
+
+If you're self-hosting Retool, we’ve created an importable app that you can use to explore Audit Logs and discover what specific users are doing on the platform, when specific actions were triggered in an app, and much more.
+
+This tool should help you investigate the activity of specific users and/or usage of specific apps on the Retool platform. 
+
+<img src="./images/audit_log_explorer.png" width=50% height=50%>
+
+## Setup
+This app needs read access to [Retool's storage database](https://docs.retool.com/docs/configuring-retools-storage-database). Therefore, it's only suitable for use with self-hosted instances of Retool. 
+
+> :warning: **For a smooth app import experience, we recommend upgrading to Retool v2.113 or above.**
+
+### 0. Make sure you have access!
+
+You should have the right approvals and know where to look to find the `POSTGRES_HOST`, `POSTGRES_PORT`, `POSTGRES_DB`, `POSTGRES_USER`, `POSTGRES_PASSWORD` environment variables in your deployment configuration. 
```

Comment:
What should the user do if they don't know where to look to find these environment variables?